src/edu/wright/cs/raiderplanner/controller/SettingsController.java

```diff
@@ -0,0 +1,412 @@
+	//chat variables
+	private final BorderPane mainPane = new BorderPane();
+
+	/**
+	 * Sets this.current to equal passed variable and calls this.main().
+	 */
+	public void main(Window wind) {
+		this.current = wind;
+		this.main();
+	}
+
+	/**
+	 * Main method containing switch statements.
+	 * Switch statements detect what state the app is in.
+	 */
+	public void main() {
+		if (isNavOpen) {
+			openMenu.fire();
+		}
+
+		initialLoad = false;
+
+		switch (this.current) {
+		case ABOUT: {
+			loadAbout();
+			break;
+		}
+		case GENERAL: {
+			this.loadGeneral();
+			break;
+		}
+		case ACCOUNT: {
+			this.loadAccount();
+			break;
+		}
+		case THEME: {
+			this.loadTheme();
+			break;
+		}
+		case NOTIFICATIONS: {
+			this.loadNotification();
+			break;
+		}
+		default:
+			break;
+		}
+	}
+
+	/**
+	 * Display the About menu.
+	 */
+	public void loadAbout() {
+		// set ToolTips
+		this.mainContent.getChildren().remove(1, this.mainContent.getChildren().size());
+		this.topBox.getChildren().clear();
+		this.title.setText("About");
+
+		//Text Labels:
+		Label appName = new Label("RaiderPlanner");
+		appName.setFont(Font.font("Ariel", FontWeight.BOLD , 42));
+
+		Label versionNo = new Label("Version 1.0.1\nCopyright © 2017");
+		versionNo.setFont(Font.font("Ariel", 12));
+		versionNo.setTextFill(Color.GRAY);
+		versionNo.setTextAlignment(TextAlignment.CENTER);
+
+		Label details = new Label("\"The best study planner since the Gantt Diagram\"");
+		details.setFont(Font.font("Ariel", FontPosture.ITALIC , 24));
+
+		Label stayOnTrack = new Label("Stay on track.");
+		stayOnTrack.setFont(Font.font("Ariel", FontWeight.BOLD, 18));
+
+		Label summary = new Label("Life gets complicated, and keeping track of homework and exam\n"
+				+ "dates is not the first thing on your mind. RaiderPlanner will be "
+				+ "there to help you out.");
+		summary.setFont(Font.font("Ariel", 18));
+		summary.setTextAlignment(TextAlignment.CENTER);
+
+		Label credits = new Label("Created By:\n"
+				+ "Roberto Sánchez\n"
+				+ "Wright State University's CEG3120 class\n\n"
+				+ "Based on the 'PearPlanner'\n"
+				+ "Created By: \n"
+				+ "Benjamin Dickson\n"
+				+ "Andrew Odintsov\n"
+				+ "Zilvinas Ceikauskas\n"
+				+ "Bijan Ghasemi Afshar\n");
+		credits.setFont(Font.font("Ariel", 14));
+		credits.setTextAlignment(TextAlignment.CENTER);
+
+		VBox detailsBox = new VBox(5);
+		details.setWrapText(true);
+		detailsBox.getChildren().addAll(
+				appName,
+				versionNo,
+				details,
+				stayOnTrack,
+				summary,
+				credits);
+		detailsBox.setAlignment(Pos.TOP_CENTER);
+
+		GridPane.setVgrow(detailsBox, Priority.SOMETIMES);
+		GridPane.setHgrow(detailsBox, Priority.ALWAYS);
+		GridPane.setColumnSpan(detailsBox, GridPane.REMAINING);
+
+		this.mainContent.addRow(2, detailsBox);
+		this.mainContent.setVgap(10);
+		this.mainContent.setPadding(new Insets(15));
+		this.mainContent.setAlignment(Pos.CENTER);
+	}
+
+	/**
+	 * Display the general settings.
+	 * NOT IMPLEMENTED
```

Comment:
I don't think we should keep the `NOT IMPLEMENTED` text. A better approach would be mark with with `TODO`. For example: `TODO: Display the general settings.` That causes Eclipse to automatically add it to the project task and highlight it with a blue marker. Same for the other occurrences of `NOT IMPLEMENTED`.

Reply:
I'll do that instead.